docs/core/additional-tools/dotnet-coverage.md

```diff
@@ -80,10 +80,6 @@ dotnet-coverage merge
 
   Removes all input coverage reports that were merged.
 
-* **`-r, --recursive`**
-
-  Search for coverage reports in subdirectories.
```

Comment:
I would keep lines 83-86, and change line 85 to this:

```
  [.NET 7 SDK and earlier versions only] Search for coverage reports in subdirectories.
```

Reply:
Updated.